src/main.js

```diff
@@ -1 +1,30 @@
-console.log('Hello, world!');
+// console.log('Hello, world!');
+// const total = document.querySelector('.form__total');
+// console.log(total);
+// // total.textContent = '';
+// const radio = document.querySelectorAll('input[type="radio"]');
+// console.log(radio);
+// // console.log(radio.checked === true);
+// function radioPayment() {
+//   if (radio.checked === true) {
+//     radio.textContent = '$300';
+//     return true;
+//   }
+// }
+// radio.forEach(radio => radio.addEventListener('change', radioPayment));
+// // radio.forEa.addEventListener('click', radioPayment());
+const amount = document.querySelector('.form__total span');
+console.log(amount);
+const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
+
+const annual = document.querySelector('input[value="Annually"]');
+const monthly = document.querySelector('input[value="Monthly"]');
```

Comment:
No need for these if you use my suggestion above...

Reply:
hahaha yeah these are from my first attempt and forgot to comment out them